web/crux-ui/src/routes.ts

```diff
@@ -531,6 +531,46 @@ class StorageRoutes {
   details = (id: string) => `${this.root}/${id}`
 }
 
+// config bundle
+
+class ConfigBundleApi {
+  private readonly root: string
+
+  constructor(root: string) {
+    this.root = `/api${root}`
```

Comment:
Is this good?  `/api${root}`

Reply:
It should be OK, because root comes from TeamRoutes which is `/teamSlug`, so it'll be `/api/teamSlug/....`.

Reply:
Ahh, okay just looks weird a little, resolved my comment.